core/store/migrations/migration1597695690/migrate.go

```diff
@@ -0,0 +1,166 @@
+package migration1597695690
+
+import (
+	"github.com/jinzhu/gorm"
+)
+
+// Migrate creates the offchain_reporting_job_specs table
+func Migrate(tx *gorm.DB) error {
+	return tx.Exec(`
+        CREATE TABLE offchainreporting_key_bundles (
+            -- NOTE: Key bundle ID is intended to be set by software as sha256 hash of {onchain sig pubkey, offchain sig pubkey, config decryption pubkey}
+            id bytea NOT NULL PRIMARY KEY,
+            CONSTRAINT chk_id_length CHECK (octet_length(id) = 32),
+            encrypted_priv_key_bundle jsonb NOT NULL,
+            created_at timestamptz NOT NULL
+        );
+
+        CREATE INDEX idx_offchainreporting_oracle_specs_unique_key_bundles_created_at ON offchainreporting_key_bundles USING BRIN (created_at);
+
+        -- NOTE: This will be extended with new IDs when we bring directrequest and fluxmonitor under the new jobspawner umbrella
+        -- Only ONE id should ever be present
+        CREATE TABLE jobs (
+            id SERIAL PRIMARY KEY
+        );
+
+        CREATE TABLE offchainreporting_oracle_specs (
+            id SERIAL PRIMARY KEY,
+            job_id INT NOT NULL REFERENCES jobs (id),
```

Comment:
This is the wrong way around

Reply:
It's currently both ways around.  See the `ALTER TABLE` below (had to move that after the creation of both tables as it was throwing an error)